content/working_with/console/widgets/blueprints.md

```diff
@@ -61,10 +61,10 @@ The default visibility is "Tenant", and according to the logged-in user's permis
    `Blueprint name` is the name with which you want to identify this blueprint once uploaded.
 5. Select `Terraform version` (by default, one of the Terraform versions will be selected).
 6. Provide `Blueprint main information`, which consists of:
-   - `URL to a zip archive that contains the Terraform module`.
-   - `Terraform folder in the archive` - selectable field from which you can select Terraform module contained in the zip file.<br />
-   By default this field is disabled, until the `URL to a zip archive that contains the Terraform module` is provided.
-   - (Optional) Credentials required for accessing `URL to a zip archive that contains the Terraform module`.
+   - `Terraform module source - URL to a zip archive or a GIT repository`.
```

Comment:
```suggestion
   - `Terraform module source` - URL to a zip archive or a Git repository.
```

Reply:
That modification looks great, but `Terraform module source - URL to a zip archive or a GIT repository` is the whole field label, which Micheal wanted to have on the modal (yup, with the capital `GIT` 😁 )

And because of that I think that it should stay as it is, WDYT?

Reply:
I also commented on 'GIT' here: https://github.com/cloudify-cosmo/cloudify-stage/pull/1907, so if that other comment is addressed then it should be updated here as well.
I understand it's a whole label in the UI, but semantically it's a label with a description, so I think it's better to just reference the actual label here. With you current implementation it looks like a formatting issue IMO:
![image](https://user-images.githubusercontent.com/49753069/161756748-e80781c4-d908-455b-94ba-7b635a415db0.png)

Reply:
Hi, it seems like you've not submitted comments/review in [cloudify-cosmo/cloudify-stage#1907](https://github.com/cloudify-cosmo/cloudify-stage/pull/1907), as I can't see any comment/note
So it's hard for me to relate to that part

But the whole point is valid 💪 
Thanks for the explaination and in that case I'll wait for resolving the comment in the [separate PR](https://github.com/cloudify-cosmo/cloudify-stage/pull/1907)

Reply:
As answered on one of the discussions on the [linked PR](https://github.com/cloudify-cosmo/cloudify-stage/pull/1907), it was confirmed that text should be indeed `Git`, not `GIT` 🎉 

Because of that, I'm applying the suggestion😉